code/modules/reagents/reagent_containers/food/drinks/juicebox.dm

```diff
@@ -1,11 +1,58 @@
 /obj/item/chems/food/drinks/juicebox
+	icon = 'icons/obj/juicebox.dmi'
+	icon_state = "juicebox_base"
+	blend_mode = BLEND_OVERLAY
 	name = "juicebox"
+	gender = PLURAL
 	desc = "A small cardboard juicebox. Cheap and flimsy."
 	volume = 30
 	amount_per_transfer_from_this = 5
 	atom_flags = 0
 	matter = list(MATERIAL_CARDBOARD = 30)
 
+	var/juiceIcon = 'icons/obj/juicebox.dmi'
```

Comment:
Any particular reason not to use the `icon` of the object rather than declaring a second var?